### ISSUE TITLE
Low accuracy for generated image from my own image using project_image.py

### ISSUE THREAD

**Issue by ymzlygw** [rolux/stylegan2encoder#22]
Thanks for your  work!
I want to know why I got the not very good generated image for my own image.
I take a photo using my phone and try to get the dlatents. I follow the guide and run the 'align_images.py' and 'project_images.py' , here is the result:
#### Origin image:
![z](https://user-images.githubusercontent.com/17065855/100058715-2743cd80-2e6d-11eb-9727-b047abc79e8a.jpeg)
#### After project_image.py:
![z](https://user-images.githubusercontent.com/17065855/100058789-4b9faa00-2e6d-11eb-87c6-0dc71971be5f.png)
#### I don't think this two image are the same person. Could you please tell me why I generated the result from real-face image?

Another question is that some time when I use 'align_images.py', but there is no generated image in 'align', and then  I can't use 'project_images.py'  to get the dlatent of image(.jpg)

**Comment by cyremur** [comment]
This happens because the model generated from the FFHD is not proficient enough to recreate every face 100% faithfully. There is no real solution to this instead of trying a couple times and getting something close enough. The only remedy would be training a better model with a better face dataset that includes more images similar to the one you want to generate.

**Comment by ymzlygw** [comment]
> This happens because the model generated from the FFHD is not proficient enough to recreate every face 100% faithfully. There is no real solution to this instead of trying a couple times and getting something close enough. The only remedy would be training a better model with a better face dataset that includes more images similar to the one you want to generate.

Thanks for your reply! 
Another problem is that every time when I want to align a image of mine, I got a image with blurred edges instead,as follows:
Origin image:
![z02e](https://user-images.githubusercontent.com/17065855/100168290-53f6f400-2f04-11eb-8da8-bdd49d19e1e2.jpg)
aligned image:
![z02ex](https://user-images.githubusercontent.com/17065855/100168303-5b1e0200-2f04-11eb-97b6-03c30c70e524.png)
It  always happens when I input a photo of mine that took by phone.Could you please tell me how to solve it?thanks

**Comment by cyremur** [comment]
This happens because the align script tries to fit your face into a square with face landmarks at certain positions. Your selfie is too close, therefore the original picture doesn't cover the entire square. I assume the artifacts on the side are created by the align algorithm trying to expand your picture to cover that square.

TL;DR: Try taking the picture from a bit further away with more background.

**Comment by ymzlygw** [comment]
> This happens because the align script tries to fit your face into a square with face landmarks at certain positions. Your selfie is too close, therefore the original picture doesn't cover the entire square. I assume the artifacts on the side are created by the align algorithm trying to expand your picture to cover that square.
> 
> TL;DR: Try taking the picture from a bit further away with more background.

Wow, Thanks a lot! I have been troubled by this problem for a long time。I'll try it next.
